forte/data/readers/ag_news_reader.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The reader that reads AG News data into Datapacks.
+"""
+from typing import Iterator, Tuple
+
+from forte.common.configuration import Config
+from forte.common.resources import Resources
+from forte.data.data_pack import DataPack
+from forte.data.readers.base_reader import PackReader
+from ft.onto.ag_news import Title, Description
+from ft.onto.base_ontology import Document
+
+__all__ = [
+    "AGNewsReader",
+]
+
+
+class AGNewsReader(PackReader):
+    r""":class:`AGNewsReaderr` is designed to read in AG News
+    text classification dataset.
```

Comment:
Include the links and citations for this dataset, so people know where to find them. You can also include how to run the reader on the downloaded data (which directory, file to pass into the reader, for example.)